xml/sec_images_hardened.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook51-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<sect1 xml:id="sec-images-hardened" xml:lang="en"
+ xmlns="http://docbook.org/ns/docbook" version="5.1"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ 
+ <title>Hardened Images</title>
+ <para>
+  To improve overall security, &suse; provides hardened images of some products.
+  The images are hardened using <phrase role="product">&openscap;</phrase>, a
+  collection of open source tools that implement the <systemitem
+  class="protocol">Security Content Automation Protocol (SCAP)</systemitem>
+  maintained by the <orgname>National Institute of Standards and Technology
+  (NIST)</orgname>. <phrase role="product">&openscap;</phrase> supports
+  automated configuration, vulnerability and patch checking, technical control
+  compliance activities, and security measurement.
+ </para>
+ <para>
+  All images are pre-hardened to the extent they can safely be hardened without
+  causing problems in public cloud frameworks. After instance creation, users
+  can use the installed <package>openscap</package> packages to complete the
+  image hardening process using the following profiles:
```

Comment:
I very minor wording change to "using the following" 
how about "using any of the following..." or "using one of the following..."

Reply:
Thanks, I incorporated your suggestion in 11f7fa2.